### PR TITLE
scale the marketing logo correct for standalone apks in landscape mode

### PR DESCRIPTION
**Pull request by manuels87**

### CODE REVIEW

**Comment by 84n4n4** [comment]
test this please

**Comment by 84n4n4** [comment]
is there a ticket for this? if so please have a look at https://confluence.catrob.at/display/KNOWHOW/Commit+Message+Guidelines

File: catroid/src/main/java/org/catrobat/catroid/ui/MarketingActivity.java [hunk 1]
```diff
@@ -123,10 +125,15 @@ public void onClick(View v) {
 		});
 	}
 
-	private Bitmap scaleDrawable2Bitmap() {
+	private Bitmap scaleDrawable2Bitmap(Boolean landscapemode) {
```
Comment (robertpainsi) [inline catroid/src/main/java/org/catrobat/catroid/ui/MarketingActivity.java]: Hate `NullPointerExceptions`? me too! **`b`**`oolean`

**Comment by manuels87** [comment]
just made the improvements

**Pull request merged**